### PR TITLE
Added a new conclusion state to support flutter autosubmit bot

### PR DESCRIPTION
**Pull request by ricardoamador**
The flutter team needs a new state to support skipping stable tests in presubmit pull requests.

### CODE REVIEW

File: lib/src/common/model/checks.dart [hunk 1]
```diff
@@ -45,6 +45,7 @@ class CheckRunConclusion extends EnumWithValue {
   static const neutral = CheckRunConclusion._('neutral');
   static const cancelled = CheckRunConclusion._('cancelled');
   static const timedOut = CheckRunConclusion._('timed_out');
+  static const skipped = CheckRunConclusion._('skipped');
```
Comment (keyonghan) [inline lib/src/common/model/checks.dart:48]: Let's add a test to cover the `skipped` case: https://github.com/SpinlockLabs/github.dart/blob/master/test/unit/checks_test.dart.

**Review by robrbecker — APPROVED** [review]

**Comment by robrbecker** [comment]
Looks good. Let me know if you want it merged or if there's more to add/review on your end.

**Comment by robrbecker** [comment]
Released in version 9.3.0 https://github.com/SpinlockLabs/github.dart/releases/tag/9.3.0

**Comment by CaseyHillers** [comment]
FYI @robrbecker I manually triggered https://github.com/SpinlockLabs/github.dart/actions/workflows/release_unreleased_prs.yml to get this to release. I think the action is misconfigured to run on the forks instead of the upstream, which is causing failures

**Comment by CaseyHillers** [comment]
9.3.0 was successfully prepped, but the last step failed with publishing. I think it's a quick upstream fix in the pub publish action in https://github.com/sakebook/actions-flutter-pub-publisher.

I've gone ahead and manually published this @ricardoamador so you should be able to use your changes

**Comment by ricardoamador** [comment]
Cool! Thank you @CaseyHillers appreciate the help.

**Pull request merged**